SpatialBatchNormalization.lua

```diff
@@ -182,3 +182,13 @@ function BN:accGradParameters(input, gradOutput, scale)
       self.gradBias:add(scale, self.buffer2) -- sum over mini-batch
    end
 end
+
+function BN:clearState()
+   self.buffer = nil
```

Comment:
The buffers have changed name with https://github.com/torch/nn/pull/550. I think you should clean now [`save_mean` and `save_std`](https://github.com/torch/nn/blob/master/SpatialBatchNormalization.lua#L76-L77)